Vegeting/Global/UIComponent/GroupCategoryView.swift

```diff
@@ -25,20 +25,27 @@ final class GroupCategoryView: UIView {
         collectionView.backgroundColor = .clear
         collectionView.showsHorizontalScrollIndicator = false
         collectionView.register(GroupCategoryCollectionViewCell.self, forCellWithReuseIdentifier: GroupCategoryCollectionViewCell.className)
+        collectionView.dataSource = self
+        collectionView.delegate = self
         return collectionView
     }()
     
     private var categoryList: [String] = ["맛집", "행사", "파티", "기타"]
     weak var delegate: GroupCategoryViewDelegate?
+    private var selectedCategory: String?
     
```

Comment:
p1;
이전에 반영한대로, 해당 View는 특정 변수에 따라서 두가지로 구분되거나 달라지는 형태가 아닌, 선택이 미리 되었냐 안되었냐의 차이기 때문에 특정 값이 존재한다면 그것만 select하는게 좋을거 같습니다~! 
```suggestion
func selectAlreadyCategory(selectedCategory: String) {
        guard let index = categoryList.firstIndex(of: selectedCategory) else { return }
        let indexPath = IndexPath(item: index, section: 0)
        categoryCollectionView.selectItem(at: indexPath, animated: true, scrollPosition: .bottom)
    }
```

Reply:
좋아요. entryPoint에 따라 꼭 받아야할 필요가 없는 친구이기 때문에 init으로 받지 않고 함수로 넘겨주는 게 좋겠군요![8d95f61](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/164/commits/8d95f613edbbde680f435a2a33b3d594df9ef022)에서 반영완료!